src/gui/pages/record.tsx

```diff
@@ -174,11 +225,24 @@ export default function Record(props: RecordeProps) {
           {uiSpec !== null && type !== null && uiSpec['visible_types'][0] !== ''
             ? '' + uiSpec.viewsets[type]['label'] + ' Record ' + hrid
             : ''}{' '}
+          {draft_id !== undefined && ' Draft '}
         </Typography>
         <Typography variant={'subtitle1'} gutterBottom>
           Edit data for this record. If you need to, you can also revisit
-          previous revisions.
+          previous revisions and resolve conflicts.
         </Typography>
+        {conflicts !== null &&
+          conflicts['available_heads'] !== undefined &&
+          conflicts['available_heads'].length > 1 && (
+            <Alert
+              severity="warning"
+              action={<ConflictHelpDialog type={'info'} />}
+              icon={<InfoOutlinedIcon />}
+            >
+              This Record has {conflicts['available_heads'].length} conflicting
+              instances. Resolve these conflicts before continue
```

Comment:
"Resolve these conflicts before Editing to avoid creating more conflicts."

Maybe this should be a modal that needs the user's action. i.e., choose from "Edit anyway?" and "Resolve conflicts"